### ISSUE TITLE
[Android] The system message does not wrap to the next line if it's too long in a group chat

### ISSUE THREAD

**Issue by mariia-skrypnyk** [status-im/status-react#21477]
# Bug Report

### Reproduction

1. Login to the app
2. Make sure you have =<2 Contacts
3. Create a group chat with an autofilled name

#### Actual behavior

![Image](https://github.com/user-attachments/assets/b4478948-1161-48d9-bef6-1e9280d39f80)

#### Expected behavior

System message wraps into two lines if it doesn't fit

### Additional Information

- Status version: nightly 22/10
- Operating System: Android

[comment]: # (Please, add logs/notes if necessary)

**Comment by churik** [comment]
I don't think that it is feasible to close this issue in 2.32, given the desired release cut date.
Deferring the issue, please be free to move it back if you think there is a room for it @ilmotta
